app/src/main/java/com/zulip/android/util/FabRemoveOnSnackbar.java

```diff
@@ -0,0 +1,47 @@
+package com.zulip.android.util;
+
+import android.support.design.widget.CoordinatorLayout;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.view.ViewCompat;
+import android.view.View;
+
+import java.util.List;
+
+/**
+ * Created by nitish on 5/2/17.
```

Comment:
Author details are saved in the git commit author info, you can remove this one!
 Add a comment that what actually this class is doing.